src/item/ToolTier.php

```diff
@@ -47,6 +48,7 @@ enum ToolTier{
 	case IRON;
 	case DIAMOND;
 	case NETHERITE;
+	case MACE;
```

Comment:
It should be obvious MACE is not a tier which can be applied to existing tools in the game.